packages/base-types/src/node/card.ts

```diff
@@ -28,3 +28,8 @@ export interface Node extends BaseNode, NodeNextID {
   type: NodeType.CARD;
   card: Card;
 }
+
+export interface TraceFramePayload extends Card {}
+export interface TraceFrame extends BaseTraceFrame<TraceFramePayload> {
+  type: TraceType.CARD;
+}
```

Comment:
do we need `Card` trace? 
I guess we are transforming `card` into `cardV2` node in the alexa/google prototype clone handlers and `cardv1` trace will never be used, isn't it?